pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -1005,6 +1008,52 @@ func (r *ReconcileHyperConverged) ensureKubeVirtTemplateValidator(instance *hcov
 	return r.client.Status().Update(context.TODO(), instance)
 }
 
+func newKubeVirtStorageRoleForCR(cr *hcov1alpha1.HyperConverged, namespace string) *rbacv1.Role {
+	labels := map[string]string{
+		"app": cr.Name,
+	}
+	return &rbacv1.Role{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "hco.kubevirt.io:config-reader",
+			Labels:    labels,
+			Namespace: namespace,
+		},
+		Rules: []rbacv1.PolicyRule{
+			{
+				APIGroups: []string{""},
+				Resources: []string{"configmaps"},
+				ResourceNames: []string{"kubevirt-storage-class-defaults"},
+				Verbs: []string{"get", "watch", "list"},
```

Comment:
can you please run ` go fmt pkg/controller/hyperconverged/hyperconverged_controller.go` and commit back the result?